Gemfile.jruby-3.1.lock.release

```diff
@@ -849,7 +850,7 @@ GEM
     tzinfo-data (1.2024.2)
       tzinfo (>= 1.0.0)
     unicode-display_width (2.6.0)
-    uri (1.0.3)
+    uri (0.12.3)
```

Comment:
hm., it is a patch update and downgrading?! 🤔 
this is something we need to check, revert would be safer.